### PR TITLE
add Q3 progress report

### PR DESCRIPTION
**Pull request by pbalcer**
This change is [Reviewable](https://reviewable.io/reviews/pmem/pmem.github.io/66)

### CODE REVIEW

File: _posts/2017-10-05-progress-report-q3-17.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+- There's an active and ongoing effort of porting the linux-specific
+parts of the library to FreeBSD, mostly led by [@gaweinbergi](https://github.com/gaweinbergi).
+Our code is naturally very linux centric due to being developed
+alongside the kernel support for NVDIMMs, and there are quite a few
+hurdles to overcome, but everything seems to be going along quite nicely.
+- Patch that ports the low-level library (libpmem) to ARM has been
+submitted to our repository [@vvenkates27](https://github.com/vvenkates27).
+This is also a place where there's a lot of platform specific code due to
+the novelty of the topic and the fact that the library was implemented
+together with the programming model for x86.
```
Comment (tomaszkapela) [inline _posts/2017-10-05-progress-report-q3-17.md]: I'd rephrase. This sentence suggests the programming model was designed for x86 and it wasn't. The SNIA doc doesn't mention any specific arch AFAIK.

File: _posts/2017-10-05-progress-report-q3-17.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+Our initial approach focuses on letting the application terminate when a `SIGBUS`
+is encountered, and then provide a set of off-line tools that allow seamless
+recovery of data from replicas.
+
+The work is still in progress and the relevant PR is [here](https://github.com/pmem/nvml/pull/2246).
+
+### Conclusion
+
+Vacation season might appear to have taken the toll this time around, but that's
```
Comment (tomaszkapela) [inline _posts/2017-10-05-progress-report-q3-17.md]: taken a toll

File: _posts/2017-10-05-progress-report-q3-17.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+mostly just that - an appearance. 
+
+The vast majority of changes this quarter were fixes and tiny improvements.
+
+For more exciting developments, look at [pmemkv](https://github.com/pmem/pmemkv), [pynvm](https://github.com/pmem/pynvm) or [pmse](https://github.com/pmem/pmse) where a lot of activity has been
```
Comment (tomaszkapela) [inline _posts/2017-10-05-progress-report-q3-17.md]: ... where there has been a lot of activity?

File: _posts/2017-10-05-progress-report-q3-17.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+concentrated around.
+
+Until the next time.
```
Comment (tomaszkapela) [inline _posts/2017-10-05-progress-report-q3-17.md]: I think it's _Until next time_.

Comment (pbalcer) [reply _posts/2017-10-05-progress-report-q3-17.md]: Hm, is using this idiom on its own ever grammatically correct? I got rid of it entirely.

**Comment by krzycz** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 4 unresolved discussions.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmem.github.io/66#-:-Kw4jRuzd14jI5Ab7AkL:bnfp4nl)*

**Comment by pbalcer** [comment]
bump
